ALLHUBS-Jan-2024-Liftoff/code-stars
# 11 star ratings

In this branch, I:
- Created StarInput component for users to submit star ratings on the review form
- Wrote updateAverageRating in ReviewService.java to calculate average campground rating on review submission
- Updated the data type of Campground.rating to Double
- Implemented DisplayRating component in Campground page and ResultTable component

To test it:
- Create a new campground
- Submit a review for the new campground. Check database to ensure average value is stored in campground.rating
- Check search page and campground page to ensure stars are rendering accurately
- Submit one or two more reviews, checking database and website after each one
